### DISCUSSION TITLE
Why is there a 2 streamers limit?

### DISCUSSION THREAD

**Discussion by ertigo** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#205] [Q&A]
I notice that the miner only watches up to 2 streams at a time? Is there any way to increase the number of streamers to watch?

**Comment by Signko** [comment]
[Here is the answer](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/5)

FYI It took me 30 seconds to find the information you were looking for by the way, not trying to be rude but sometimes a little bit of research helps.
